src/lib/db/DatabaseManager.ts

```diff
@@ -9,84 +9,159 @@ import { v4 as uuidv4 } from "uuid";
+    },
+    {
+      id: uuidv4(),
+      name: "Change Status",
+      timestamp: new Date().toISOString(),
+    },
+  ],
+  toolkit: [
+    {
+      id: uuidv4(),
+      name: "Listen to my favourite music",
+      categories: ["Replace", "Barrier"],
+      checked: false,
+      infoUrl: "https://google.com/music",
+      imageUrl:
+        "https://daily.jstor.org/wp-content/uploads/2023/01/good_times_with_bad_music_1050x700.jpg",
+      timestamp: new Date().toISOString(),
+    },
+    {
+      id: uuidv4(),
+      name: "Watch TV",
+      categories: ["Distract"],
+      checked: false,
+      infoUrl: "https://google.com/tv",
+      imageUrl:
+        "https://daily.jstor.org/wp-content/uploads/2023/01/good_times_with_bad_music_1050x700.jpg",
+      timestamp: new Date().toISOString(),
+    },
+    {
+      id: uuidv4(),
+      name: "Call a friend",
+      categories: ["Distract", "Change status"],
+      checked: false,
+      infoUrl: "https://example.com/call",
+      imageUrl:
+        "https://daily.jstor.org/wp-content/uploads/2023/01/good_times_with_bad_music_1050x700.jpg",
+      timestamp: new Date().toISOString(),
+    },
+    {
+      id: uuidv4(),
+      name: "See a friend",
+      categories: ["Distract", "Change status"],
+      checked: false,
+      infoUrl: "https://example.com/call",
+      imageUrl:
+        "https://daily.jstor.org/wp-content/uploads/2023/01/good_times_with_bad_music_1050x700.jpg",
+      timestamp: new Date().toISOString(),
+    },
+  ],
+};
 
 class DatabaseManager {
   dbInstance: RxDatabase | null = null;
 
   async initialiseDatabase() {
+    console.groupCollapsed("Initialising Database");
     if (!this.dbInstance) {
+      console.groupCollapsed("Creating New Database");
       this.dbInstance = await createRxDatabase({
         name: "database",
         //ignoreDuplicate: true,
         storage: getRxStorageDexie(),
       });
+      console.log("Database Created");
+      console.groupEnd();
 
+      console.groupCollapsed("Adding Collections");
       await this.dbInstance.addCollections({
         categories: { schema: categoriesSchema },
         mood_records: { schema: moodRecordSchema },
         toolkit_items: { schema: toolkitItemSchema },
       });
+      console.log("Collections Added");
+      console.groupEnd();
 
-      console.log("Database initialised");
-      await this.seedToolkitItems(); // Automatically seed data
+      console.groupCollapsed("Seeding Database");
+      await this.seedCategories();
+      await this.seedToolkitItems();
+      console.log("Database Seeded");
+      console.groupEnd();
     }
+    console.groupEnd();
     return this.dbInstance;
   }
+  
+  async seedCategories() {
+    console.groupCollapsed("Seeding Categories");
+    try {
+      console.log("Looking for database");
+      const db = await this.initialiseDatabase();
+      if (!db) {
+        console.error("Database initialisation failed");
+        return;
+      }
```

Comment:
```suggestion
    try {
      let db;
      if (this.dbInstance) {
        console.log("Database found");
        db = this.dbInstance;
      } else {
        console.log("Looking for database");
        db = await this.initialiseDatabase();
        if (!db) {
          console.error("Database initialisation failed");
          return;
        }     
      }
```

I think this should work, and would avoid unnecessarily re-running initialiseDb

Reply:
Move `console.log("Looking for database");` back up to just below `let db;` and I'll gladly accept it

Reply:
Done, edited original suggestion on both